zones/rainverse.wiki

```diff
@@ -25,7 +25,14 @@ $ORIGIN rainverse.wiki.
 ; Servers
 
 ; Services
+shrink		CNAME	blankeclair.github.io.
+downscale	A	45.79.218.85
+		AAAA	2600:3c02::f03c:92ff:fe46:06b6
```

Comment:
Thanks for the PR. I admit I'm not a DNS syntax expert but are you sure you don't need to add 'downscale' here and that the AAA record under the A record works like this? cc @Universal-Omega

Reply:
No, I don't think so. I mainly copied what I saw, which was:
* https://en.wikipedia.org/wiki/Zone_file#Example_file
* https://github.com/miraheze/dns/blob/36c13d36f710a74cb3a5b9d3e4e3d8fb2a666b86/zones/miraheze.org#L21-L24